mycroft/configuration/mycroft.ini

```diff
@@ -93,3 +93,10 @@ max_time = 600  # in seconds
 notify_delay = 5  # in seconds
 rate = 16000
 channels = 1
+
+[PhillipsHueSkill]
+verbose = False
```

Comment:
I'd like to keep this in, if you're not strongly against it. I live in a small studio apartment, so for me, the lights changing is all the feedback I need. Having Mycroft tell me what I can see him doing is just noise - it's how the Amazon Echo works, and I hate it (trying to turn off the lights before bed results in a jarring 'OK!' any time the volume is left too high).

Someone with a larger home, however, may appreciate the feedback, if, for example, they are trying to turn off the living room lights from the bedroom.